lib/bgm.dart

```diff
@@ -48,11 +48,11 @@ class Bgm extends WidgetsBindingObserver {
   ///
   /// It is safe to call this function even when a current BGM track is
   /// playing.
-  Future<void> play(String filename, {double volume}) async {
+  Future<void> play(String filename, {double? volume}) async {
     volume ??= 1;
```

Comment:
move the `??=` as a default value for a non-null parameter. volume=null makes no sense

Reply:
done